src/pkg/services/m365/api/lists.go

```diff
@@ -29,11 +30,18 @@ const (
 
 	ContentTypeColumnDisplayName = "Content Type"
 
-	AddressFieldName     = "address"
-	CoordinatesFieldName = "coordinates"
-	DisplayNameFieldName = "displayName"
-	LocationURIFieldName = "locationUri"
-	UniqueIDFieldName    = "uniqueId"
+	AddressFieldName          = "address"
+	NestedCityFieldName       = "city"
+	NestedCountryFieldName    = "countryOrRegion"
+	NestedPostalCodeFieldName = "postalCode"
+	NestedStateFieldName      = "state"
+	NestedStreetFieldName     = "street"
+	CoordinatesFieldName      = "coordinates"
+	NestedLatitudeFieldName   = "latitude"
+	NestedLongitudeFieldName  = "longitude"
+	DisplayNameFieldName      = "displayName"
+	LocationURIFieldName      = "locationUri"
+	UniqueIDFieldName         = "uniqueId"
```

Comment:
this is getting a bit difficult to read, both here in the declaration and in usage.  I think you'd be okay to use some abbreviations to simplify "FieldName" to FN.  However, like stated in other comments, I think you're using the term `fieldName` in place of `key`, as in the key for a map entry.  I think we should avoid that vocabulary, since it's more difficult to grok while `key` is very simple.
```suggestion
	AddressKey          = "address"
	CoordinatesKey      = "coordinates"
	DisplayNameKey      = "displayName"
	LocationURIKey      = "locationUri"
	UniqueIDKey         = "uniqueId"
	
	// entries that are nested within a second layer
	CityKey       = "city"
	CountryKey    = "countryOrRegion"
	PostalCodeKey = "postalCode"
	StateKey      = "state"
	StreetKey     = "street"
	LatitudeKey   = "latitude"
	LongitudeKey  = "longitude"
```